Event.swift

```diff
@@ -36,7 +36,37 @@ class Event: PFObject, PFSubclassing {
     @NSManaged var amountPerAttendee: Int
     @NSManaged var location: String?
     @NSManaged var startTime: NSDate?
+    @NSManaged var endTime: NSDate?
+    var coverImageUrl: NSURL?
     
+    var moneyCollected: Int? {
+        return nil
+    }
+    
+    var totalMoneyToCollect: Int? {
+        return nil
+    }
+    
+}
+
+func <(lhs: Event, rhs: Event) -> Bool {
+    if let lhsStartTime = lhs.startTime {
+        if let rhsStartTime = rhs.startTime {
```

Comment:
Fixed.

I initially wanted to divide those `if`s to better control intermediate state for comparison, but it came out not so readable, so I refactored it.